### ISSUE TITLE
Spawn of pumpkin spiders

### ISSUE THREAD

**Issue by uhloin** [DivineRPG/DivineRPG#517]
DivineRPG-1.9.5 for 1.19
In 1.19 all hostile mobs do not spawn at places with light > 0 but pumpkin spiders do it.

**Comment by NicosaurusRex99** [comment]
We do want some hostile mobs to respect light levels.

**Comment by velocityraptor-real** [comment]
most divinerpg mobs are monsters, which follow the vanilla minecraft monster spawn rule, which prevents them from spawning with light > 0

**Comment by uhloin** [comment]
I have cyclops spawning under roof in my house and pumpkin spiders and cobblers inside fenced area with light. Cyclops are peaceful but like cocroaches :) May be carpets will help against them. 

It's common rule in almost any game that home is safe place where player can relax. Breaking of some traditions and stereotypes makes the game more interesting and challenging but in other cases more annoying. I think mobs in "safe zone" are second case. There should be no respawn or some simple item to ward off them.

**Comment by velocityraptor-real** [comment]
I recommend the torchmaster mod for completeley removing mob spawns in a larger radius

**Comment by Ringedtech203** [comment]
I thought that they could only spawn on grass, was that removed? They shouldnt spawn in your house unless it had a grass floor

**Comment by uhloin** [comment]
velocityraptor-real
Thank you for workaround with torchmaster, it works with some tweaks. I had to set "aggressive" mode in its config or else dreed lantern does not affect cyclops. Also I added a golem to exceptions list to keep golem farm working.

Ringedtech203
You're right wiki says they should spawn on the grass only but they spawned in the house. First floor has oak logs, second floor oak slabs (upper half-block). May be enderman bring some grass and take it later...
